kpops/components/streams_bootstrap/base.py

```diff
@@ -96,3 +96,11 @@ def manifest_deploy(self) -> tuple[KubernetesManifest, ...]:
             )
 
         return resource
+
+    @override
+    def manifest_destroy(self) -> tuple[KubernetesManifest, ...]:
+        return (
+            tuple(StrimziKafkaTopic.from_topic(topic) for topic in self.to.kafka_topics)
+            if self.to
+            else ()
+        )
```

Comment:
I would prefer early return pattern to make this a little easier to read

```suggestion
    if not self.to:
        return ()
    return tuple(StrimziKafkaTopic.from_topic(topic) for topic in self.to.kafka_topics)
```

also what about `super()` call?